models/weather_api_handler.py

```diff
@@ -0,0 +1,74 @@
+#!usr/bin/python3
+"""
+module to handle all incoming request for weather information
+"""
+
+import requests
+from models.exceptions import *
+import os
+
+
+class WeatherRequestHandler:
+    """handle API for weather"""
+
```

Comment:
This URL could just be requestUrl or something, then in the init, we could call (set_request_url) that will use the information passed in to set if it is forecast or future.